### PR TITLE
Custom border radius for inputs and general clean ups

### LINKED ISSUE #121: Should not use the generic `$border-radius` for form inputs in 'Core -> Base -> Forms'

### PR DESCRIPTION
**Pull request by chris-pearce**
This closes #121 & #122.

### CODE REVIEW

**Comment by AlecRaeside** [comment]
LGTM

File: core/base/_forms.scss [hunk 1]
```diff
@@ -10,6 +10,34 @@
...
+
+// Border
+$text-input-and-select-border-thickness:      1 !default;
+
+$text-input-and-select-border-style:          solid !default;
+
+$text-input-and-select-border-colour:         lighten($colour-black, 30%)
+!default;
+
+$text-input-and-select-border-radius-amount:  $border-radius !default;
```
Comment (kllevin) [inline core/base/_forms.scss]: When you were referring to a similar setting with buttons you named the variable: `$c-btn-rounded-corners-size` do you want to use something similar?

**Comment by kllevin** [comment]
other than that one question.... LGTM

File: core/base/_forms.scss (hunk 1 above)
Comment (chris-pearce) [inline core/base/_forms.scss]: Yes most definitely! I'm started to get a bit worried about the amount of setting names in trying to keep them consistent + needing the option to set a name-space.

**Comment by kllevin** [comment]
LGTM

**Pull request merged**